CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
-cmake_minimum_required(VERSION 2.8.4 FATAL_ERROR)
+cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
```

Comment:
do all the changes still work in debian jessie?

Reply:
Yup, just tested it using a Docker image.